src/wallet/sqlite.cpp

```diff
@@ -206,7 +206,9 @@ void SQLiteDatabase::Open()
     }
 
     if (m_db == nullptr) {
-        TryCreateDirectories(m_dir_path);
+        if (!m_mock) {
+            TryCreateDirectories(m_dir_path);
+        }
```

Comment:
in commit d52f502b1ea1cafa7d58c5517f01dba26ecb7269:

Could this commit include a description, so that the patch can be understood by non-wallet people as well?